jung-algorithms/src/main/java/edu/uci/ics/jung/algorithms/util/KMeansClusterer.java

```diff
@@ -161,6 +161,15 @@ public void setConvergenceThreshold(double convergence_threshold) {
     return clusterMap.values();
   }
 
+  private static double[] meansOf(Collection<double[]> collectionOfDoubleArrays) {
```

Comment:
You could even consider just inlining this in the code, since in this context you only call it once.

Reply:
Done. :)